cmd/influxd/config.go

```diff
@@ -251,6 +246,38 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Address       string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          uint16 `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeparator string `toml:"name-separator"`
+}
+
+// Default carbon port per http://graphite.readthedocs.org/en/1.0/feeding-carbon.html
+const defaultGrahitePort = 2004
+
+// ConnnectionString returns the connection string for this Graphite config in the form
+// host:port.
+func (g Graphite) ConnectionString(defaultBindAddr string) string {
```

Comment:
I'm always torn about making a pointer receiver if we don't mutate (just a personal nit of mine I know).  Is it important that we make them all pointer receivers just to be consistent?  This debate comes up all the time at the go meetups I run and I know it is a very divided topic.  In general, if we pic a stance I'm fine with it as long as we all follow the same convention.